packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -122,15 +122,54 @@ exports.onCreateNode = ({ node, getNode, actions }, pluginOptions) => {
     path.resolve('src/releases')
   );
   if (isReleaseNotesPage) {
-    const releaseNoteSlug = trimTrailingSlash(slug) || '/';
+    let releaseNoteSlug = generateReleaseNoteSlug(
+      node.frontmatter.date,
+      node.frontmatter.title,
+      node.fileAbsolutePath
+    );
+    releaseNoteSlug = trimTrailingSlash(releaseNoteSlug) || '/';
```

Comment:
Let's just move the `trimTrailingSlash(releaseNoteSlug) || '/'` part into the `generateReleaseNoteSlug` function